src/lib/types/Order.ts

```diff
@@ -147,6 +147,7 @@ export interface Order extends Timestamps {
 		product: Product;
 		quantity: number;
 		customPrice?: { amount: number; currency: Currency };
+		customProductName?: string;
```

Comment:
Same, variation ids should be stored.

When creating an order, if the variations ids from the cart are no longer existing, it should throw an error or remove items from the cart